src/components/views/rooms/wysiwyg_composer/utils/message.ts

```diff
@@ -71,9 +76,51 @@ export async function sendMessage(
     }*/
     PosthogAnalytics.instance.trackEvent<ComposerEvent>(posthogEvent);
 
-    const content = await createMessageContent(message, isHTML, params);
+    let content: IContent | null = null;
 
-    // TODO slash comment
+    // Functionality here approximates what can be found in SendMessageComposer.sendMessage()
+    if (message.startsWith("/") && !message.startsWith("//")) {
+        const { cmd, args } = getCommand(message);
+        if (cmd) {
+            // we will need to handle /me separately, see SlashCommands.tsx:1387
```

Comment:
It seems a little brittle to refer to a line number like this, is it possible to just describe the location?

Reply:
Yep the action is already documented as it's what is used by the current composer. Fair point on the comment, I couldn't figure out a way to make it a useful link so I've just described the location more generally.